Extensions/Debugger/JsExtension.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+    createExtension: function(_, gd) {
+      const extension = new gd.PlatformExtension();
+      extension.setExtensionInformation(
+        'Debugger',
```

Comment:
I wonder if we should call this extension `DebuggerTools`? 
`Debugger` seems a bit too broad, in the sense that this is not this extension that is bringing its own debugger. Instead, it's using the debugger interface that is exposed by the game engine. In a codebase, I would have been calling methods exposed by this extension "helpers", "tools", etc.. so I think `DebuggerTools` could be a better name (and easier to grep in the codebase).